apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/docs/web/OfficeBrandIconsImplementation.md

```diff
@@ -1,8 +1,26 @@
 To use the Microsoft 365 multicolor brand icons, select the format and size that best meets your needs. Fluent UI includes a media query that automatically selects the right image file for the pixel density of the screen you’re targeting.
 
-The following code shows you how to specify a product icon by extension, item type, icon size, and image type using Fluent UI's `<Icon>` component:
+The following code shows you how to specify a 96px product icon by brand using the office-ui-fabric-core css and a `<div>` element:
 
 ```jsx
-// Sample code for displaying an Word 96x96px Icon
-<div class="ms-BrandIcon--icon96 ms-BrandIcon--word" />
+// Sample code for using office-ui-fabric-core version 11.0.0 to display an Word 96x96px Icon
+<link
+  rel="stylesheet"
+  href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css"
+/>
+
+<div class="ms-BrandIcon--icon96 ms-BrandIcon--word"></div>
+```
+
+This following code shows you how to specify a 48px product icon by brand using the office-ui-fabric-core SVG and a Fluent UI `<Image>` component (a regular `<img>` element can be used too):
+
+```jsx
+import { Image } from '@fluentui/react';
```

Comment:
what's the benefit of using `Image` here? `img` should be sufficient. I feel samples should be as simple as possible.

Reply:
When I first tried with `<img>` it was rendering as the size of the whole page, and it seemed like I would need some extra CSS to get it to render properly, and our brand icon example internally uses Image so I went that direction. 

However, I just tried again per your comment and this works just fine, I'll update the example. 
```jsx
<img 
  src="https://static2.sharepointonline.com/files/fabric/assets/brand-icons/product/svg/word_48x1.svg" 
  width="48"
  height="48"
  alt="Word product icon" 
/>
```

Reply:
Fixed, thanks!